code/datums/spells/wizard_spells.dm

```diff
@@ -190,6 +190,29 @@
 /obj/effect/proc_holder/spell/area_teleport/teleport/create_new_targeting()
 	return new /datum/spell_targeting/self
 
+/obj/effect/proc_holder/spell/return_to_teacher
+	name = "Return to Teacher"
+	desc = "This spell teleports you back to your teacher."
+
+	school = "abjuration"
+	base_cooldown = 30 SECONDS
+	clothes_req = TRUE
+	invocation = "SCYAR TESO"
+	invocation_type = "shout"
+	cooldown_min = 10 SECONDS //5 Second reduction per rank
```

Comment:
Why is there a comment on reduction on an apprentice spell? It's technically fine, but I don't see this being made publically available in the near future

Reply:
I mean its not necessary, I just followed the standards of other wiz spells without thinking about it
Also for sacred flame, it's fine that the apprentice can light the teacher on fire, it's part of the risk you take buying a fire based apprentice